build.sc

```diff
@@ -317,10 +319,10 @@ trait MillPublishJavaModule extends MillJavaModule with PublishModule {
 /**
  * Some custom scala settings and test convenience
  */
-trait MillScalaModule extends ScalaModule with MillJavaModule { outer =>
+trait MillScalaModule extends ScalaModule with MillJavaModule with ScalafixModule{ outer =>
   def scalaVersion = Deps.scalaVersion
   def scalacOptions =
-    super.scalacOptions() ++ Seq("-deprecation", "-P:acyclic:force", "-feature", "-Xlint:unused")
+    super.scalacOptions() ++ Seq("-deprecation", "-P:acyclic:force", "-feature", "-Ywarn-unused:imports")
```

Comment:
What's the different between `-Xlint:unused` and `-Ywarn-unused`?

Reply:
Turns out that it's the same, I reverted to the old one